uplink/metainfo/client.go

```diff
@@ -73,11 +62,22 @@ func NewClient(ctx context.Context, tc transport.Client, address string, apiKey
 		return nil, Error.Wrap(err)
 	}
 
-	return &Metainfo{client: pb.NewMetainfoClient(conn)}, nil
+	return &Client{
+		client: pb.NewMetainfoClient(conn),
+		conn:   conn,
+	}, nil
+}
+
+// Close closes the dialed connection.
+func (client *Client) Close() error {
```

Comment:
Is this method used somewhere except tests? I cannot find where it closes the connection in production code

Reply:
I added TODO notes for bunch of things that should be closing this and `lib/uplink.Project.Close` calls this.